src/scss/_elements.layout.scss

```diff
@@ -1,6 +1,33 @@
+html,
+body {
+  height: 100%;
+}
+
 body {
   margin: 0;
   background-color: $black;
   color: $white;
   font-family: $font-body;
 }
+
+body > div {
+  position: relative;
+
+  @supports (mix-blend-mode: overlay){
+    &::before {
+      content: '';
+      display: block;
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      width: 100%;
+      background-image: url('/images/noise.png');
```

Comment:
This background isn't applying to the whole page, it stops at different points depending on screen size. Here, it's stopping at the unsuccessful design system chart:

![Screen Shot 2019-07-10 at 10 22 42 AM](https://user-images.githubusercontent.com/23404383/60977033-bb56d880-a2fc-11e9-858e-19477336f26c.png)

Reply:
This is specific to Chrome. Ugh. And the blend-mode looks awful in the browser. Why do people use this browser?!